api/api/spec/spec.yaml

```diff
@@ -5324,7 +5324,7 @@ components:
         </tr>
         <tr>
         <td>monitor</td>
-        <td>internal</td>
+        <td>reports</td>
```

Comment:
`/monitor/internal` still exists, this `reports` should be added to the list but not replacing `internal`:

> `GET /manager/configuration/monitor/internal`
```JSON
{
  "data": {
    "affected_items": [
      {
        "monitord": {
          "day_wait": 10,
          "compress": 1,
          "sign": 1,
          "monitor_agents": 1,
          "keep_log_days": 31,
          "rotate_log": 1,
          "size_rotate": 536870912,
          "daily_rotations": 12,
          "delete_old_agents": 0
        }
      }
    ],
    "total_affected_items": 1,
    "total_failed_items": 0,
    "failed_items": []
  },
  "message": "Active configuration was successfully read in specified node",
  "error": 0
}
```

Reply:
`/monitor/internal` appears twice in the reference. This modification changes the one that returns the content of the tag `<reports>`